source/TipsterText.cpp

```diff
@@ -71,8 +73,10 @@ TipsterText::MouseDown(BPoint point)
 	GetMouse(&temp, &buttons);
 
 	if (Bounds().Contains(temp)) {
-		BMessage message(UPDATE_TIP);
-		
-		fMessenger->SendMessage(&message);
+		if (buttons == 1)  //left mouse button
```

Comment:
Insted of a comment, you can use the constants B_PRIMARY_MOUSE_BUTTON and B_SECONDARY_MOUSE_BUTTON.